device/mellanox/x86_64-mlnx_msn2700-r0/platform_reboot

```diff
@@ -28,6 +28,17 @@ function SafePwrCycle() {
 
 ParseArguments "$@"
 
+# Reboot immediately if the kdump capture kernel is running
+VMCORE_FILE=/proc/vmcore
+if [ -e $VMCORE_FILE -a -s $VMCORE_FILE ]; then
+    sync ; sync
+    umount -fa > /dev/null 2&>1
```

Comment:
I believe this is `2>&1`. Do you have test output?

Reply:
fixed. test output [test.log](https://github.com/sonic-net/sonic-buildimage/files/12179397/test.log)